packages/babel-jest/src/index.js

```diff
@@ -20,7 +20,8 @@ import fs from 'fs';
 import path from 'path';
 import jestPreset from 'babel-preset-jest';
 import {transform as babelTransform, util as babelUtil} from 'babel-core';
-import babelIstanbulPlugin from 'babel-plugin-istanbul';
+
+const getBabelPluginIstanbul = () => require('babel-plugin-istanbul').default;
```

Comment:
for this, we can pass `require.resolve('babel-plugin-istanbul')`, no need for us to require that module ourselves